local_planner/src/nodes/local_planner_node.cpp

```diff
@@ -111,9 +111,6 @@ LocalPlannerNode::~LocalPlannerNode() { delete server_; }
 void LocalPlannerNode::readParams() {
   // Parameter from launch file
   auto goal = local_planner_.getGoal();
-  nh_.param<double>("goal_x_param", goal.x, 9.0);
```

Comment:
can you remove the parameters `goal_*_param` from the launch files

Reply:
Done. We need to keep the z-parameter though, since this would break the compatibility with RViz: the goal set when using the 2D Nav Goal button always has a z-coordinate of 0, which is why we don't want to overwrite it in its callback (`clickedGoalCallback`)